README.md

# MultData

<!-- badges: start -->
<!-- badges: end -->

The goal of MultData is to provide ready-to-use datasets for teaching and learning multivariate data analysis techniques.

Absolutely no claims are made over the ownership of the data included in this package.

The datasets were found either:

1. on the internet (somewhere), or
2. among the files left by former staff on a shared network drive. 

There is no guarantee that data in these files is a perfect match for the printed data. Use them at your own risk.

## Installation

You can install the development version of MultData from [GitHub](https://github.com/) with:

``` r
# install.packages("devtools")
devtools::install_github("ajrgodfrey/MultData")
```



NEWS.md
# MultData 0.0.2

# MultData 

- adding data from Manly 1994 2nd edition; still used in 4th edition published in 2017
- adding Johnson and Wichern data


* Initial CRAN submission.


R/ManlyData.R
### Data used in Manly texts 1986, 1994 (primary reference), and most recently in 2017
##   BannadiBurials, Belize, BumpusSparrows, ButterflyColonies, EgyptianSkulls, EuroEmp, EuroFood, Goblets, Mandible, RoadsNZ, Steneryd, VotingNJ

#'
#' Bannadi Burials Data from Manly (1994)
#'
#' Table 9.4 on page 143-144 of Manly (1994)
#'
#' @rdname BannadiBurials
#' @name BannadiBurials
#' @format 
#'     A data frame with 47 observations on 40 variables.
#'  \describe{
#'   \item{\code{Burial}}{a character vector}
#'   \item{\code{Body.Type}}{a numeric vector}
#'   \item{\code{X1}}{a numeric vector}
#'   \item{...}{...}
#'   \item{\code{X38}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage BannadiBurials
NULL

#'
#' Belize Soil Data from Manly (1994)
#'
#' Table 10.2 on page 158  of Manly (1994)
#'
#' @rdname Belize
#' @name Belize
#' @format 
#'     A data frame with 151 observations on the following eight variables.
#'  \describe{
#'   \item{\code{X1}}{a numeric vector}
#'   \item{\code{X2}}{a numeric vector}
#'   \item{\code{X3}}{a numeric vector}
#'   \item{\code{X4}}{a numeric vector}
#'   \item{\code{Y1}}{a numeric vector}
#'   \item{\code{Y2}}{a numeric vector}
#'   \item{\code{Y3}}{a numeric vector}
#'   \item{\code{Y4}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage Belize
NULL

#'
#' Bumpus Sparrows Data from Manly (1994)
#'
#' Table 1.1 on page 2 of Manly (1994)
#'
#' @rdname BumpusSparrows
#' @name BumpusSparrows
#' @format 
#'     A data frame with 49 observations on the following six variables.
#'  \describe{
#'   \item{\code{Survivorship}}{a character vector}
#'   \item{\code{Total_length}}{a numeric vector}
#'   \item{\code{Alar_extent}}{a numeric vector}
#'   \item{\code{L_beak_head}}{a numeric vector}
#'   \item{\code{L_humerous}}{a numeric vector}
#'   \item{\code{L_keel_sternum}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage BumpusSparrows
NULL

#'
#' Butterfly Colonies Data from Manly (1994)
#'
#' Table 1.3 on page 8 of Manly (1994)
#'
#' @rdname ButterflyColonies
#' @name ButterflyColonies
#' @format 
#'     A data frame with 16 observations on the following 10 variables.
#'  \describe{
#'   \item{\code{Altitude}}{a numeric vector}
#'   \item{\code{Ann.Prec}}{a numeric vector}
#'   \item{\code{Max.Temp}}{a numeric vector}
#'   \item{\code{Min.Temp}}{a numeric vector}
#'   \item{\code{PFPGi0.40}}{a numeric vector}
#'   \item{\code{PFPGi0.60}}{a numeric vector}
#'   \item{\code{PFPGi0.80}}{a numeric vector}
#'   \item{\code{PFPGi1.00}}{a numeric vector}
#'   \item{\code{PFPGi1.16}}{a numeric vector}
#'   \item{\code{PFPGi1.30}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage ButterflyColonies
NULL




#'
#' Egyptian Skulls Data from Manly (1994)
#'
#' Table 1.2 on page 4 of Manly (1994)
#'
#' @rdname EgyptianSkulls
#' @name EgyptianSkulls
#' @format 
#'     A data frame with 150 observations on the following five variables.
#'  \describe{
#'   \item{\code{Period}}{a character vector}
#'   \item{\code{Maximum.breadth}}{a numeric vector}
#'   \item{\code{Basibregmatic.height}}{a numeric vector}
#'   \item{\code{Basialveolar.length}}{a numeric vector}
#'   \item{\code{Nasal.height}}{a numeric vector}
#' }
#' @source also found as dataset No. 361 in Hand et al. (1994)
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage EgyptianSkulls
NULL

#'
#' European Employment Data from Manly (1994)
#'
#' Table 1.5 on page 11 of Manly (1994)
#'
#' @rdname EuroEmp
#' @name EuroEmp
#' @format 
#'     A data frame with thirty observations on the following eleven variables.
#'  \describe{
#'   \item{\code{Country}}{a character vector}
#'   \item{\code{Group}}{a character vector}
#'   \item{\code{AGR}}{a numeric vector}
#'   \item{\code{MIN}}{a numeric vector}
#'   \item{\code{MAN}}{a numeric vector}
#'   \item{\code{PS}}{a numeric vector}
#'   \item{\code{CON}}{a numeric vector}
#'   \item{\code{SER}}{a numeric vector}
#'   \item{\code{FIN}}{a numeric vector}
#'   \item{\code{SPS}}{a numeric vector}
#'   \item{\code{TC}}{a numeric vector}
#' }
#' @source also found as dataset No. 363 in Hand et al. (1994)
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage EuroEmp
NULL

#'
#' European Food Consumption Data from Manly (1994)
#'
#' Table 6.7 on page  90 of Manly (1994)
#'
#' @rdname EuroFood
#' @name EuroFood
#' @format 
#'     A data frame with 25 observations on the following ten variables.
#'  \describe{
#'   \item{\code{X}}{a character vector}
#'   \item{\code{RedMeat}}{a numeric vector}
#'   \item{\code{WhiteMeat}}{a numeric vector}
#'   \item{\code{Eggs}}{a numeric vector}
#'   \item{\code{Milk}}{a numeric vector}
#'   \item{\code{Fish}}{a numeric vector}
#'   \item{\code{Cereals}}{a numeric vector}
#'   \item{\code{Starch}}{a numeric vector}
#'   \item{\code{Nuts}}{a numeric vector}
#'   \item{\code{FruitVeg}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage EuroFood
NULL




#'
#' Goblets Data from Manly (1994)
#'
#' Table 6.6 on page 89  of Manly (1994)
#'
#' @rdname Goblets
#' @name Goblets
#' @format 
#'     A data frame with 25 observations on the following six variables.
#'  \describe{
#'   \item{\code{InrWth}}{a numeric vector}
#'   \item{\code{BwlWth}}{a numeric vector}
#'   \item{\code{TtlHt}}{a numeric vector}
#'   \item{\code{BaseWth}}{a numeric vector}
#'   \item{\code{StmWth}}{a numeric vector}
#'   \item{\code{StmHt}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage Goblets
NULL


#'
#' Mandible Data from Manly (1994)
#'
#' Table 1.4 on page 10 of Manly (1994)
#'
#' @rdname Mandible
#' @name Mandible
#' @format 
#'     A data frame with observations on the following variables.
#'  \describe{
#' }
#' @source also found as dataset No. 358 in Hand et al. (1994)
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage Mandible
NULL


#'
#' Road Distances between NZ Towns Data from Manly (1994)
#' 
#'Table 11.1 on page 173 of Manly (1994)
#'
#' @rdname RoadsNZ
#' @name RoadsNZ
#' @format 
#'     A data frame with distances between Alexandra, Balclutha, Blenheim, Christchurch, Dunedin, Franz.Josef, Greymouth, Invercargill, Milford, Nelson, Queenstown, Te.Anau, and Timaru.
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage RoadsNZ
NULL

#'
#' Steneryd Reserve Plants Data from Manly (1994)
#'
#' Table 9.3 on page 142 of Manly (1994)
#'
#' @rdname Steneryd
#' @name Steneryd
#' @format 
#'     A data frame with 25 observations on the following 17 variables.
#'  \describe{
#'   \item{\code{X1}}{a numeric vector}
#'   \item{...}{...}
#'   \item{\code{X17}}{a numeric vector}
#' }
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage Steneryd
NULL



#'
#' Voting of New Jersey Data from Manly (1994)
#'
#' Table 11.3 on page 177 of Manly (1994)
#'
#' @rdname VotingNJ
#' @name VotingNJ
#' @format 
#'     A data frame with 15 observations on the following 15 variables.
#'  \describe{
#'   \item{\code{Hunt.R}}{a numeric vector}
#'   \item{\code{Sandman.R}}{a numeric vector}
#'   \item{\code{Howard.D}}{a numeric vector}
#'   \item{\code{Thompson.D}}{a numeric vector}
#'   \item{\code{Frelinghuysen.R}}{a numeric vector}
#'   \item{\code{Forsythe.R}}{a numeric vector}
#'   \item{\code{Widnall.R}}{a numeric vector}
#'   \item{\code{Roe.D}}{a numeric vector}
#'   \item{\code{Helstoski.D}}{a numeric vector}
#'   \item{\code{Rodino.D}}{a numeric vector}
#'   \item{\code{Minish.D}}{a numeric vector}
#'   \item{\code{Rinaldo.R}}{a numeric vector}
#'   \item{\code{Maraziti.R}}{a numeric vector}
#'   \item{\code{Daniels.D}}{a numeric vector}
#'   \item{\code{Pattern.D}}{a numeric vector}
#' }
#' @source also found as dataset No. 294 in Hand et al. (1994)
#' @references Manly, B.F.J. (1994) \emph{Multivariate Statistical Methods: A Primer}, Second Edition, CRC Press.
#' @author contributed by A. Jonathan R. Godfrey
#' @usage VotingNJ
NULL


R/JohnsonWichern.R
# data from Johnson and Wichern (2007) Applied Multivariate Statistical Analysis, 6th edition

## tab1.10 

'data.frame':	76 obs. of  9 variables:
 $ Breed    : int  1 1 1 1 1 1 1 1 1 1 ...
 $ SalePrice: int  2200 2250 1625 4600 2150 1225 2250 4000 1600 1525 ...
 $ Yheight  : num  51 51.9 49.9 53.1 51.2 49.2 51 51.5 50.1 49.6 ...
 $ FFBody   : int  1128 1108 1011 993 996 985 959 1060 979 1083 ...
 $ FFBPrct  : num  70.9 72.1 71.6 68.9 68.6 71.4 72.1 69.3 71.2 75.8 ...
 $ Frame    : int  7 7 6 8 7 6 7 7 6 6 ...
 $ BackFat  : num  0.25 0.25 0.15 0.35 0.25 0.15 0.2 0.3 0.25 0.3 ...
 $ SHeight  : num  54.8 55.3 53.1 56.4 55 51.4 54 55.6 51.5 54.6 ...
 $ SWeight  : int  1720 1575 1410 1595 1488 1500 1522 1765 1365 1640 ...

## tab1.2 

'data.frame':	41 obs. of  3 variables:
 $ Density: num  0.801 0.824 0.841 0.816 0.84 0.842 0.82 0.802 0.828 0.819 ...
 $ Machine: num  121 128 129 132 135 ...
 $ Cross  : num  70.4 72.5 78.2 74.9 71.2 ...

## tab1.3 

'data.frame':	25 obs. of  3 variables:
 $ Mass    : num  5.53 10.4 9.21 8.95 7.06 ...
 $ Body    : num  59 75 69 67.5 62 62 74 47 86.5 69 ...
 $ HindLimb: num  114 142 124 125 130 ...

## tab1.5 

'data.frame':	42 obs. of  7 variables:
 $ Wind  : int  8 7 7 10 6 8 9 5 7 8 ...
 $ SolarR: int  98 107 103 88 91 90 84 72 82 64 ...
 $ CO    : int  7 4 4 5 4 5 7 6 5 5 ...
 $ NO    : int  2 3 3 2 2 2 4 4 1 2 ...
 $ NO2   : int  12 9 5 8 8 12 12 21 11 13 ...
 $ O3    : int  8 5 6 15 10 12 15 14 11 9 ...
 $ HC    : int  2 3 3 4 3 4 5 4 3 4 ...

## tab1.6 

'data.frame':	98 obs. of  6 variables:
 $ Age: int  18 19 20 20 20 21 21 21 22 22 ...
 $ S1T: num  152 138 144 144 149 ...
 $ S1D: num  1.6 0.4 0 3.2 0 0.8 1.6 1.6 3.2 0 ...
 $ S2T: num  198 181 186 195 218 ...
 $ S2D: num  0 1.6 0.8 0 0 0.8 0.8 3.2 3.2 0.2 ...
 $ MS : int  0 0 0 0 0 0 0 0 0 0 ...

## tab1.7 

'data.frame':	98 obs. of  6 variables:
 $ Symptoms    : num  0.889 2.813 1.454 0.294 2.727 ...
 $ Activity    : num  1.389 1.437 1.091 0.941 2.545 ...
 $ Sleep       : num  1.555 0.999 2.364 1.059 2.819 ...
 $ Eat         : num  2.22 2.31 2.46 2 2.73 ...
 $ Appetite    : num  1.95 2.31 2.91 1 4.09 ...
 $ SkinReaction: int  1 2 3 1 0 1 2 1 1 1 ...

## tab1.8 

'data.frame':	25 obs. of  6 variables:
 $ DomRadius : num  1.103 0.842 0.925 0.857 0.795 ...
 $ Radius    : num  1.052 0.859 0.873 0.744 0.809 ...
 $ DomHumerus: num  2.14 1.87 1.89 1.74 1.73 ...
 $ Humerus   : num  2.24 1.74 1.81 1.55 1.72 ...
 $ DomUlna   : num  0.873 0.59 0.767 0.706 0.549 0.782 0.737 0.618 0.853 0.823 ...
 $ Ulna      : num  0.872 0.744 0.713 0.674 0.654 0.571 0.803 0.682 0.777 0.765 ...

## tab1.9 

'data.frame':	54 obs. of  8 variables:
 $ X       : chr  "ARG" "AUS" "AUT" "BEL" ...
 $ S100    : num  11.6 11.1 11.2 11.1 11.5 ...
 $ S200    : num  22.9 22.2 22.7 22.5 23.1 ...
 $ S400    : num  52.5 48.6 50.6 51.5 53.3 ...
 $ M800    : num  2.05 1.98 1.94 1.97 2.07 1.97 1.97 2 1.93 2.04 ...
 $ M1500   : num  4.25 4.02 4.05 4.08 4.29 4.17 4 4.22 3.84 4.34 ...
 $ M3000   : num  9.19 8.63 8.78 8.82 9.81 9.04 8.54 9.26 8.1 9.37 ...
 $ Marathon: num  150 144 154 143 174 ...

## tab11.10 

'data.frame':	66 obs. of  4 variables:
 $ Sex : chr  "Female" "Female" "Female" "Female" ...
 $ Age : int  2 2 2 2 2 2 2 2 2 2 ...
 $ Tail: int  127 171 171 164 165 127 162 133 173 145 ...
 $ Body: int  441 455 462 446 463 393 451 376 475 398 ...

## tab11.2 

'data.frame':	100 obs. of  4 variables:
 $ Origin: int  1 1 1 1 1 1 1 1 1 1 ...
 $ Gender: int  2 1 1 2 1 2 1 2 2 1 ...
 $ Fresh : int  108 131 105 86 99 87 94 117 79 99 ...
 $ Marine: int  368 355 469 506 402 423 440 489 432 403 ...

## tab11.4 

'data.frame':	46 obs. of  5 variables:
 $ CF.TD      : num  -0.45 -0.56 0.06 -0.07 -0.1 -0.14 0.04 -0.07 0.07 -0.14 ...
 $ NI.TA      : num  -0.41 -0.31 0.02 -0.09 -0.09 -0.07 0.01 -0.06 -0.01 -0.14 ...
 $ CA.CL      : num  1.09 1.51 1.01 1.45 1.56 0.71 1.5 1.37 1.37 1.42 ...
 $ CA.NS      : num  0.45 0.16 0.4 0.26 0.67 0.28 0.71 0.4 0.34 0.43 ...
 $ NotBankrupt: int  0 0 0 0 0 0 0 0 0 0 ...

## tab11.7 

'data.frame':	56 obs. of  6 variables:
 $ V        : num  3.9 2.7 2.8 3.1 3.5 3.9 2.7 5 3.4 1.2 ...
 $ Fe       : num  51 49 36 45 46 43 35 47 32 12 ...
 $ Be       : num  0.2 0.07 0.3 0.08 0.1 0.07 0 0.07 0.2 0 ...
 $ Saturated: num  7.06 7.14 7 7.2 7.81 6.25 5.11 7.06 5.82 5.54 ...
 $ Aromatic : num  12.2 12.2 11.3 13 12.6 ...
 $ Location : chr  "Wilhelm" "Wilhelm" "Wilhelm" "Wilhelm" ...

## tab11.8 

'data.frame':	75 obs. of  3 variables:
 $ Group   : int  1 1 1 1 1 1 1 1 1 1 ...
 $ Activity: num  -0.0056 -0.1698 -0.3469 -0.0894 -0.1679 ...
 $ Antigen : num  -0.1657 -0.1585 -0.1879 0.0064 0.0713 ...

## tab11.8b 

'data.frame':	10 obs. of  2 variables:
 $ Activity: num  -0.112 -0.059 0.064 -0.043 -0.05 -0.094 -0.123 -0.011 -0.21 -0.126
 $ Antigen : num  -0.279 -0.068 0.012 -0.052 -0.098 -0.113 -0.143 -0.037 -0.09 -0.019

## tab11.9 

'data.frame':	43 obs. of  11 variables:
 $ Product19   : chr  "ACCheerios" "Cheerios" "CocoaPuffs" "CountChocula" ...
 $ Manufacturer: chr  "G" "G" "G" "G" ...
 $ Calories    : int  110 110 110 110 110 110 110 110 100 130 ...
 $ Protein     : int  2 6 1 1 1 3 2 2 2 3 ...
 $ Fat         : int  2 2 1 1 1 1 1 1 1 2 ...
 $ Na          : int  180 290 180 180 280 250 260 180 220 170 ...
 $ Fibre       : num  1.5 2 0 0 0 1.5 0 0 2 1.5 ...
 $ Carbs       : chr  "10.5" "17" "12" "12" ...
 $ Sugar       : int  10 1 13 13 9 10 3 12 6 10 ...
 $ K           : int  70 105 55 65 45 90 40 55 90 120 ...
 $ Group       : int  1 1 1 1 1 1 1 1 1 1 ...

## tab3.2 

'data.frame':	25 obs. of  2 variables:
 $ Severity: num  12.5 14.5 8 9 19.5 8 9 7 7 9 ...
 $ Work    : num  13.7 16.5 17.4 11 23.6 13.2 32.1 12.3 11.8 24.4 ...

## tab4.3 

'data.frame':	30 obs. of  5 variables:
 $ Shockwave: int  1889 2403 2119 1645 1976 1712 1943 2104 2983 1745 ...
 $ Vibration: int  1651 2048 1700 1627 1916 1712 1685 1820 2794 1600 ...
 $ Static1  : int  1561 2087 1815 1110 1614 1439 1271 1717 2412 1384 ...
 $ Static2  : int  1778 2197 2222 1533 1883 1546 1671 1874 2581 1508 ...
 $ Distance : num  0.6 5.48 7.62 5.21 1.4 ...

## tab4.6 

'data.frame':	130 obs. of  7 variables:
 $ Independence: int  27 12 14 18 9 18 12 10 8 21 ...
 $ Support     : int  13 13 20 20 22 15 18 19 23 11 ...
 $ Benevolence : int  14 24 15 17 22 17 29 21 23 17 ...
 $ Conformity  : int  20 25 16 12 21 25 18 10 19 9 ...
 $ Leadership  : int  11 6 7 6 6 9 4 16 9 16 ...
 $ Gender      : int  2 2 2 2 2 2 2 2 2 2 ...
 $ Status      : int  1 1 1 1 1 1 1 1 1 1 ...

## tab5.1 

'data.frame':	20 obs. of  3 variables:
 $ Rate: num  3.7 5.7 3.8 3.2 3.1 4.6 2.4 7.2 6.7 5.4 ...
 $ Na  : num  48.5 65.1 47.2 53.2 55.5 36.1 24.8 33.1 47.4 54.1 ...
 $ K   : num  9.3 8 10.9 12 9.7 7.9 14 7.6 8.5 11.3 ...

## tab5.11 

'data.frame':	30 obs. of  2 variables:
 $ Stiffness: int  1232 1115 2205 1897 1932 1612 1598 1804 1752 2067 ...
 $ Strength : int  4175 6652 7612 10914 10850 7627 6954 8365 9469 6410 ...

## tab5.12 

'data.frame':	45 obs. of  2 variables:
 $ Tail: int  191 197 208 180 180 188 210 196 191 179 ...
 $ Wing: int  284 285 288 273 275 280 283 288 271 257 ...

## tab5.2 

'data.frame':	87 obs. of  3 variables:
 $ History: int  468 428 514 547 614 501 421 527 527 620 ...
 $ Verbal : int  41 39 53 67 61 67 46 50 55 72 ...
 $ Science: int  26 26 21 33 27 29 22 23 19 32 ...

## tab5.9 

'data.frame':	40 obs. of  4 variables:
 $ Voltage: num  23 22 22.8 22.1 22.5 22.2 22 22.1 22.5 22.5 ...
 $ Current: int  276 281 270 278 275 273 275 268 277 278 ...
 $ Speed  : num  290 289 288 288 288 ...
 $ Flow   : num  51 51.7 51.3 52.3 53 51 53 54 52 52 ...

## tab6.10 

'data.frame':	59 obs. of  4 variables:
 $ Fuel   : num  16.44 7.19 9.92 4.24 11.2 ...
 $ Repair : num  12.43 2.7 1.35 5.78 5.05 ...
 $ Capital: num  11.23 3.92 9.75 7.78 10.67 ...
 $ Type   : chr  "gasoline" "gasoline" "gasoline" "gasoline" ...

## tab6.11 

'data.frame':	45 obs. of  2 variables:
 $ Tail: int  180 186 206 184 177 177 176 200 191 193 ...
 $ Wing: int  278 277 308 290 273 284 267 281 287 271 ...

## tab6.12 

'data.frame':	50 obs. of  5 variables:
 $ Resting1: num  0.34 0.39 0.48 0.31 0.36 0.33 0.43 0.48 0.21 0.32 ...
 $ Resting2: num  3.71 5.08 5.13 3.95 5.51 4.07 4.77 6.69 3.71 4.35 ...
 $ MaxO1   : num  2.87 3.38 4.13 3.6 3.11 3.95 4.39 3.5 2.82 3.59 ...
 $ MaxO2   : num  30.9 43.9 44.5 46 47 ...
 $ Sex     : chr  "male" "male" "male" "male" ...

## tab6.15 

'data.frame':	70 obs. of  8 variables:
 $ Wlength  : int  85 87 94 92 96 91 90 92 91 87 ...
 $ Wwidth   : int  41 38 44 43 43 44 42 43 41 38 ...
 $ P3Length : int  31 32 36 32 35 36 36 36 36 35 ...
 $ P3Width  : int  13 14 15 17 14 12 16 17 14 11 ...
 $ P4Length : int  25 22 27 28 26 24 26 26 23 24 ...
 $ A12Length: int  9 13 8 9 10 9 9 9 9 9 ...
 $ A13Length: int  8 13 9 9 10 9 9 9 9 10 ...
 $ Species  : int  0 0 0 0 0 0 0 0 0 0 ...

## tab6.16 

'data.frame':	24 obs. of  6 variables:
 $ DomRadius : num  1.027 0.857 0.875 0.873 0.811 ...
 $ Radius    : num  1.051 0.817 0.88 0.698 0.813 ...
 $ DomHumerus: num  2.27 1.72 1.95 1.67 1.64 ...
 $ Humerus   : num  2.25 1.71 1.76 1.44 1.66 ...
 $ DomUlna   : num  0.869 0.602 0.765 0.761 0.551 0.753 0.708 0.687 0.844 0.869 ...
 $ Ulna      : num  0.964 0.689 0.738 0.698 0.619 0.515 0.787 0.715 0.656 0.789 ...

## tab6.18 

'data.frame':	36 obs. of  5 variables:
 $ W560   : num  9.33 8.74 9.31 8.27 10.22 ...
 $ W720   : num  19.1 19.6 19.2 16.4 25 ...
 $ Species: chr  "SS" "SS" "SS" "SS" ...
 $ Time   : int  1 1 1 1 2 2 2 2 3 3 ...
 $ Rep    : int  1 2 3 4 1 2 3 4 1 2 ...

## tab6.19 

'data.frame':	56 obs. of  3 variables:
 $ Length: num  271 477 306 365 466 ...
 $ Weight: num  18.5 82.5 23.4 33.5 69 ...
 $ Sex   : chr  "F" "F" "F" "F" ...

## tab6.20 

'data.frame':	16 obs. of  6 variables:
 $ Severity      : chr  "Low" "Low" "Low" "Low" ...
 $ Complexity    : chr  "Simple" "Simple" "Simple" "Simple" ...
 $ Experience    : chr  "Novice" "Novice" "Guru" "Guru" ...
 $ Assessment    : num  3 2.3 1.7 1.2 6.7 7.1 5.6 4.5 4.5 4.7 ...
 $ Implementation: num  6.3 5.3 2.1 1.6 12.6 12.8 8.8 9.2 9.5 10.7 ...
 $ Total         : num  9.3 7.6 3.8 2.8 19.3 19.9 14.4 13.7 14 15.4 ...

## tab6.5 

'data.frame':	15 obs. of  4 variables:
 $ Initial: num  87.3 59 76.7 70.6 54.9 78.2 73.7 61.8 85.3 82.3 ...
 $ Year1  : num  86.9 60.2 76.5 76.1 55.1 75.3 70.8 68.7 84.4 86.9 ...
 $ Year2  : num  86.7 60 75.7 72.1 57.2 69.1 71.8 68.2 79.2 79.4 ...
 $ Year3  : num  75.5 53.6 69.5 65.3 49 67.6 74.6 57.4 67 77.4 ...

## tab6.6 

'data.frame':	16 obs. of  4 variables:
 $ Initial: num  83.8 65.3 81.2 75.4 55.3 70.3 76.5 66 76.7 77.2 ...
 $ Year1  : num  85.5 66.9 79.5 76.7 58.3 72.3 79.9 70.9 79 74 ...
 $ Year2  : num  86.2 67 84.5 74.3 59.1 70.6 80.4 70.3 76.9 77.8 ...
 $ Year3  : num  81.2 60.6 75.2 66.7 54.2 68.6 71.6 64.1 70.3 67.9 ...

## tab6.7 

'data.frame':	60 obs. of  3 variables:
 $ Mass  : num  7.51 5.03 5.87 11.09 2.42 ...
 $ Length: num  74 69.5 72 80 56 94 95.5 99.5 97 90.5 ...
 $ Genus : int  0 0 0 0 0 0 0 0 0 0 ...

## tab6.8 

'data.frame':	32 obs. of  4 variables:
 $ Wdiff: num  869 995 1056 1126 1044 ...
 $ Wsame: num  860 875 930 954 909 ...
 $ Adiff: num  691 678 833 888 865 ...
 $ Asame: num  601 659 826 728 839 797 766 986 735 657 ...

## tab6.9 

'data.frame':	48 obs. of  4 variables:
 $ Length: int  98 103 103 105 109 123 123 133 133 133 ...
 $ Width : int  81 84 86 86 88 92 95 99 102 102 ...
 $ Height: int  38 38 42 42 44 50 46 51 51 51 ...
 $ Sex   : chr  "female" "female" "female" "female" ...

## tab7.4 

'data.frame':	63 obs. of  5 variables:
 $ Sendout: int  227 236 228 252 238 195 261 406 428 366 ...
 $ DHD    : int  32 31 30 34 28 24 39 62 62 49 ...
 $ DHDLag : int  30 32 31 30 34 28 24 39 62 62 ...
 $ Wind   : int  12 8 8 8 12 8 16 16 16 16 ...
 $ Weekend: int  1 1 0 0 0 0 0 1 1 0 ...

## tab7.5 

'data.frame':	20 obs. of  6 variables:
 $ Charge   : num  0.375 1 1 1 1.625 ...
 $ Discharge: num  3.13 3.13 3.13 3.13 3.13 3.13 3.13 5 5 5 ...
 $ Depth    : num  60 76.8 60 60 43.2 60 60 76.8 43.2 43.2 ...
 $ Temp     : int  40 30 20 20 10 20 20 10 10 30 ...
 $ Voltage  : num  2 1.99 2 1.98 2.01 2 2.02 2.01 1.99 2.01 ...
 $ Cycles   : int  101 141 96 125 43 16 188 10 3 386 ...

## tab7.7 

'data.frame':	62 obs. of  8 variables:
 $ Breaking: num  21.3 21.2 20.7 19.5 20.4 ...
 $ Elastic : num  7.04 6.98 6.78 6.6 6.79 ...
 $ Stress  : num  5.33 5.24 5.06 4.48 4.91 ...
 $ Strength: num  0.932 0.871 0.742 0.513 0.577 0.784 0.358 0.215 0.432 0.372 ...
 $ Fibre   : num  -0.03 0.015 0.025 0.03 -0.07 -0.05 -0.247 -0.099 -0.242 -0.188 ...
 $ Long    : num  35.2 35.7 39.2 39.8 33 ...
 $ Fine    : num  37 36.9 30.6 21.1 36.6 ...
 $ Tensile : num  1.06 1.06 1.05 1.05 1.05 ...

## tab8.5 

'data.frame':	61 obs. of  5 variables:
 $ Population: num  2.67 2.25 3.12 5.14 5.54 5.04 3.14 2.43 5.38 7.34 ...
 $ ProfDegree: num  5.71 4.37 10.27 7.44 9.25 ...
 $ Employed  : num  69 73 64.9 71.3 74.9 ...
 $ Govt      : num  30.3 43.3 32 24.5 31 48.2 37.6 36.8 19.7 24.5 ...
 $ HomeValue : num  1.48 1.44 2.11 1.85 2.23 1.6 1.52 1.4 2.07 1.42 ...

## tab8.6 

'data.frame':	54 obs. of  10 variables:
 $ Country : chr  "Argentina" "Australia" "Austria" "Belgium" ...
 $ S100    : num  10.23 9.93 10.15 10.14 10.27 ...
 $ S200    : num  20.4 20.1 20.4 20.2 20.3 ...
 $ S400    : num  46.2 44.4 45.8 45 45.3 ...
 $ M800    : num  1.77 1.74 1.77 1.73 1.79 1.7 1.75 1.76 1.77 1.8 ...
 $ M1500   : num  3.68 3.53 3.58 3.57 3.7 3.57 3.53 3.65 3.61 3.72 ...
 $ M5000   : num  13.3 12.9 13.3 12.8 14.6 ...
 $ M10000  : num  27.6 27.5 27.7 26.9 30.5 ...
 $ Marathon: num  130 128 132 127 146 ...
 $ X       : logi  NA NA NA NA NA NA ...

## tab8.7 

'data.frame':	76 obs. of  9 variables:
 $ Family  : int  12 54 11 21 61 20 29 29 57 23 ...
 $ Distance: num  80 8 13 13 30 70 35 35 9 33 ...
 $ Cotton  : num  1.5 6 0.5 2 3 0 1.5 2 5 2 ...
 $ Maize   : num  1 4 1 2.5 5 2 2 3 5 2 ...
 $ Sorghum : num  3 0 0 1 0 3 0 2 0 1 ...
 $ Millet  : num  0.25 1 0 0 0 0 0 0 0 0 ...
 $ Bulls   : int  2 6 0 1 4 2 0 0 4 2 ...
 $ Cattle  : int  0 32 0 0 21 0 0 0 5 1 ...
 $ Goats   : int  1 5 0 5 0 3 0 0 2 7 ...
